applications/serial_lte_modem/src/http_c/slm_at_httpc.c

```diff
@@ -337,57 +336,92 @@ static int headers_cb(int sock, struct http_request *req, void *user_data)
 	return len;
 }
 
-static int payload_cb(int sock, struct http_request *req, void *user_data)
+int do_send_payload(const uint8_t *data, int len)
+{
+	/* payload sent to server */
+	ssize_t pl_sent = 0;
+	/* payload to send to server */
+	size_t pl_to_send = 0;
+
+	if (data == NULL || len <= 0) {
+		return -EINVAL;
+	}
+
+	/* Verity payload length to be sent */
+	if (httpc.total_sent + len > httpc.pl_len) {
+		LOG_WRN("send unexpected payload");
+		pl_to_send = httpc.pl_len - httpc.total_sent;
+	} else {
+		pl_to_send = len;
+	}
+
+	/* Start to send payload */
+	while (pl_sent < pl_to_send) {
+		ssize_t ret;
+
+		ret = send(httpc.fd, data + pl_sent,
+				MIN(pl_to_send - pl_sent,
+				HTTPC_FRAG_SIZE), 0);
```

Comment:
nit: alignment
```suggestion
		ret = send(httpc.fd, data + pl_sent,
			   MIN(pl_to_send - pl_sent, HTTPC_FRAG_SIZE), 0);
```

Reply:
The code was changed per suggestion, thank you.